packages/react/src/components/Authenticator/Router/Router.tsx

```diff
@@ -16,6 +16,7 @@ const getRouteComponent = (route: string) => {
     case 'authenticated':
     case 'idle':
     case 'setup':
+    case 'autoSignIn':
```

Comment:
We might need to add this case for Vue an Angular. I noticed an unusual visual regression where after you enter your code in the confirm sign up page, there is now a floating bar at the top of the page that appears for a second or two before the authenticated page appears. It's a bit distracting. It also appears at sign out.

Screen Shot 2022-05-19 at 8.53.22 AM

Reply:
Good catch! That's been frustrating to me. I'll create a new PR that targets this branch.

Reply:
#1937 up!